internal/kubernetes/operator/deployment/deployment.go

```diff
@@ -561,3 +572,57 @@ func buildServerlessPrivateEndpoints(deploymentStore store.ServerlessPrivateEndp
 	}
 	return result, nil
 }
+
+func BuildFlexDeployments(deploymentStore store.OperatorClusterStore, projectID, projectName, clusterID, targetNamespace string, dictionary map[string]string, version string) (*akov2.AtlasDeployment, error) {
+	deployment, err := deploymentStore.FlexCluster(projectID, clusterID)
+	if err != nil {
+		return nil, err
+	}
+
+	if !isFlexExportable(deployment) {
+		return nil, nil
```

Comment:
not really blocking but i would like to see this idiom changed in a follow-up. We know this is a smell.

Reply:
We can bring this up when discussed about the ones in AKO repo, however, is widely used in CLI